liquidity/fees.go

```diff
@@ -384,3 +405,40 @@ func splitOffChain(available, prepayAmt,
 func scaleMinerFee(estimate btcutil.Amount) btcutil.Amount {
 	return estimate * btcutil.Amount(minerMultiplier)
 }
+
+func (f *FeePortion) loopInLimits(amount btcutil.Amount,
+	quote *loop.LoopInQuote) error {
+
+	// Calculate the total amount that this swap may spend in fees, as a
+	// portion of the swap amount.
+	feeLimit := ppmToSat(amount, f.PartsPerMillion)
+
+	// Check individual fee components so that we can give more specific
+	// feedback.
+	if quote.MinerFee >= feeLimit {
```

Comment:
nit: why >= ?

Reply:
Reasoning was that if we use the whole fee limit for our miner fee, we won't have anything left for our swap fee. 

That doesn't really come across very clearly, and these checks are mostly aesthetic, so will change to just `>`, since we have the full amount check later anyway

Reply:
Yeah I thought the same just wasn't sure. Thanks for explaining :+1: